package.json

```diff
@@ -9,7 +9,7 @@
     "build:ui:watch": "npm run build:ui -- --watch",
     "build:components": "echo 'Not needed anymore, just use build:ui'",
     "build:components:watch": "echo 'Not needed anymore, just use build:ui:watch'",
-    "build": "cross-env NODE_ENV=production npm run build:ui",
+    "build": "cross-env NODE_ENV=Production npm run build:ui",
```

Comment:
eh, but it should be `production` with lower case. What you are doing is basically disabling the production build...